Alley-Interactive/ruleset.xml

```diff
@@ -18,6 +18,12 @@
 	<!-- Use the VIP Go ruleset. -->
 	<rule ref="WordPress-VIP-Go" />
 
+	<!-- Use the PHPCompatibilityWP rules -->
+	<rul ref="PHPCompatibilityWP" />
+
+	<!-- Check for cross-version support for PHP 7.3 and higher. -->
+	<config name="testVersion" value="7.3-"/>
```

Comment:
Is it possible to default the Alley standards to PHP 7.4 and have the projects that _aren't_ on 7.4 yet override this in their phpcs configs?

Reply:
Yeah, I'd be happy with that. Makes sense really, since we're defaulting all our tooling to prefer PHP 7.4 everywhere else.